pkgs/games/gimx/variant.nix

```diff
@@ -0,0 +1,14 @@
+{
+  afterglow = ''
```

Comment:
Can you move this into the main nix file behind an if?

Reply:
More devices may be added in the future. I didn't want to fill `default.nix` with device specific patches.